tools/config-generator/main.go

```diff
@@ -301,9 +301,9 @@ func addMonitoringPubsubLabelsToJob(data *baseProwJobTemplateData, runID string)
 }
 
 // addVolumeToJob adds the given mount path as volume for the job.
-func addVolumeToJob(data *baseProwJobTemplateData, mountPath, name string, isSecret bool, content []string) {
+func addVolumeToJob(data *baseProwJobTemplateData, mountPath, name string, isSecret bool, isReadOnly bool, content []string) {
```

Comment:
AFAICT you aren't using readwrite volumes, is this in anticipation of something else?

Reply:
the kubeconfig is mounted to `/root/.kube`, and this dir should be readwrite, because extra files are written to it. With readonly setup, the erros occurs
```
error: open /root/.kube/config.lock: read-only file system
```

Reply:
Oh I see, the absence of readonly has made them mounted readwrite. Makes sense, could you throw a comment in somewhere with this info?

Reply:
sure, will add the comment with description

Reply:
done